crates/ruma-client-api/src/delayed_events/delayed_state_event.rs

```diff
@@ -149,12 +149,12 @@ pub mod unstable {
                 timeout: Duration::from_millis(1_234_321),
             });
             assert_eq!(
-                "https://homeserver.tld/_matrix/client/v3/rooms/!roomid:example.org/state/m.room.topic/@userAsStateKey:example.org?org.matrix.msc4140.delay=1234321",
+                "https://homeserver.tld/_matrix/client/v3/rooms/!roomid:example.org/state/com.example.custom_state/@userAsStateKey:example.org?org.matrix.msc4140.delay=1234321",
                 parts.uri.to_string()
             );
             assert_eq!("PUT", parts.method.to_string());
             assert_eq!(
-                json!({"topic": "my_topic"}),
+                json!({"key": "value"}),
```

Comment:
This check failed because of the cargo feature, but the test shouldn't even use an arbitrary  event type with an improper state key anyway, so I made it use a custom event type.

Reply:
Sure, but let's also normalize the formatting :) 

```suggestion
                json!({ "key": "value" }),
```